packages/amplify-appsync-simulator/src/server/operations.ts

```diff
@@ -12,7 +12,7 @@ import { runSubscription } from '../utils/graphql-runner/subscriptions';
 import { extractIamToken } from '../utils/auth-helpers/helpers';
 import { REALTIME_SUBSCRIPTION_PATH } from './subscription/websocket-server/server';
 
-const MAX_BODY_SIZE = '10mb';
+const MAX_BODY_SIZE = '10 MB';
```

Comment:
do we know what this const is used for ?
i.e. is it used for any kind of comparison ?

Reply:
I see it used on line 23 as a parameter to [express](http://expressjs.com/en/api.html#express.json). Given this is the style used in the docs, I think this should be reverted.

https://github.com/aws-amplify/amplify-cli/pull/11670/files#diff-34c5c7d59d82c3e54d1dcfa2c369162861dcd7c3fb953de185efbcf64bf1823dR23